Dataset: {
    name: "SYNTHIA",
}

Landmark: {
    type : "GRAPH",
    blob_filter_type: "ABSOLUTE", # or "RELATIVE"
    min_blob_size: 500,
    dilate_and_erode: true,
    num_dilate: 5,
    num_erode: 5,
    blob_neighbor_distance: 10
}

Landmark: {
    type: "ORB",
    num_features: 500
}

Landmark: {
    type: "SIFT",
    num_features: 500
}

Landmark: {
    type: "SURF",
    hessian_threshold: 0.2,
}

Landmark: {
    type: "HISTOGRAM"
}

Matcher: {
    type: "GRAPH",
    vertex_similarity_score: "WEIGHTED",  # or "SURFACE"
    random_walk_sampling_type: "UNIFORM", # or "AVOIDING"
    num_walks: 400,
    walk_length: 3
}

Matcher: {
    type: "VECTOR",
    num_retained_matches: 1
}

# Parameters read exclusively by XViewWorker

XViewWorker: {
    semantics_image_topic: "/Stereo_Left/Omni_F/labels",
    depth_image_topic: "/Stereo_Left/Omni_F/depth",
    sensor_frame: "cam00",
    world_frame: "world"
}


XViewWorker: {
    semantics_image_topic: "/Stereo_Left/Omni_R/labels",
    depth_image_topic: "/Stereo_Left/Omni_R/depth",
    sensor_frame: "cam01",
    world_frame: "world"
}

XViewWorker: {
    semantics_image_topic: "/Stereo_Left/Omni_B/labels",
    depth_image_topic: "/Stereo_Left/Omni_B/depth",
    sensor_frame: "cam02",
    world_frame: "world"
}


# Parameters read exclusively by XViewBagReader
XViewBagReader: {
  bag_file_name: "/home/carlo/polybox/Master/Master thesis/x-view/x_view_ros/x_view_node/data/Synthia_3_cams.bag",
  back: {
    semantics_image_topic: "/Stereo_Left/Omni_B/labels",
    depth_image_topic: "/Stereo_Left/Omni_B/depth",
    sensor_frame: "cam02"
  },
  right: {
    semantics_image_topic: "/Stereo_Left/Omni_R/labels",
    depth_image_topic: "/Stereo_Left/Omni_R/depth",
    sensor_frame: "cam01"
  },
  front: {
    semantics_image_topic: "/Stereo_Left/Omni_F/labels",
    depth_image_topic: "/Stereo_Left/Omni_F/depth",
    sensor_frame: "cam00"
  },

  transform_topic: "/tf",
  world_frame: "world"
}